device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/append.config.j2

```diff
@@ -1,3 +1,26 @@
+{# Construct config.bcm to include additional soc properties per specific device metadata requirement #}
+{%- set mmu_sock = 'mmu_init_config="MSFT-TH2-Tierr1"' -%}
```

Comment:
>ock = 'mmu_init_config="MSFT-TH2-Tierr1"' -%} [](start = 13, length = 45)

mmu_init_config="MSFT-TH2-Tier1"

youi have type here.

Reply:
Thanks for catching it!
Have corrected it in the new change.